databricks-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -3,7 +3,7 @@
 https://kedro.readthedocs.io/en/stable/kedro_project_setup/settings.html."""
 
 # Instantiated project hooks.
-from {{cookiecutter.python_package}}.hooks import SparkHooks
+from {{cookiecutter.python_package}}.hooks import SparkHooks # noqa: import-outside-toplevel
```

Comment:
Can you try to run this locally? I am not sure if using the name of the rule works.

We used a few linter and they are not so consistent.

`pylint` use `# pylint: disable=<rule_number> or <rule_name>` but `flake` use ` # noqa: <rule_number>`, AFAIK it doesn't work with the name but please check.

Reply:
Changed it to the rule number.

Reply:
I think I've found the error.